lib/postgres_to_redshift/update_tables.rb

```diff
@@ -30,6 +30,12 @@ def full
             end
           end
         end
+
+        keys.each do |key|
+          with_retry do
+            target_connection.exec(key.to_sql)
```

Comment:
Ah - I need to figure out a solution for when the primary or foreign key already exists.

Reply:
If it's a full import, would this be an issue?

Reply:
Yeh true, as long as it when it retries nothing is left over it will be fine.